src/app/components/papyrus/script/PapyrusScriptReference.tsx

```
@@ -11,54 +9,59 @@ import { Tooltip } from "../../tooltip/Tooltip";
             </Link>
         </Tooltip>;
     } else if (scriptAggregate) {
-        if (propsObj.inTooltip) return <span className={styles.reference}>{getBestNameVariant(scriptAggregate.name)[1]}</span>;
+        if (propsObj.inTooltip) return <span className={styles.reference}>{name}</span>;
         return <Tooltip role='tooltip' wrapperClassName={styles.reference} tooltipContents={<PapyrusScriptReferenceTooltip scriptAggregate={scriptAggregate} />}>
-            <Link href={`/${toLowerCase(game)}/script/${toLowerCase(scriptAggregate.name[0]![1])}` as const}>
-                {getBestNameVariant(scriptAggregate.name)[1]}
+            <Link href={`/${toLowerCase(game)}/script/${toLowerCase(name)}` as const}>
+                {name}
             </Link>;
         </Tooltip>;
```

Biome (1.9.4)

[error] There is a suspicious semicolon in the JSX element.

This is usually the result of a typo or some refactor gone wrong.
Remove the semicolon, or move it inside a JSX element.

(lint/suspicious/noSuspiciousSemicolonInJsx)